### DISCUSSION TITLE
In-season daily report

### DISCUSSION THREAD

**Discussion by BrianCollinss** [APSIMInitiative/ApsimX#8740] [Q&A] [answered]
What's the easiest way to have APSIM only write daily outputs if a crop is on the ground and alive? I know how to do it with a manager script. I thought there might be a way to do so via the report variables and/or events.

**Comment by hol353** [comment, marked as answer]
See the section 'Conditional reporting' in the report documentation PDF here: https://apsimnextgeneration.netlify.app/modeldocumentation/

You might be able to do something like:
```
Wheat.IsAlive == true
```

**Reply by BrianCollinss** [reply]
Thanks @hol353. Just a quick follow-up. When I use, for example, Sesame.IsAlive == true in the "Report events" panel, when is this condition evaluated? At the end/start of day, or when DoReport is called?

**Reply by hol353** [reply]
It will evaluate the expression at the end of each day of the simulation
